src/matchers/toThrowWithMessage/predicate.test.js

```diff
@@ -1,89 +0,0 @@
-import predicate from './predicate';
-
-describe('toThrowWithMessage Predicate for a string message and same type of error returns true', () => {
-  test.each`
-    error                          | type              | errorMessage
-    ${new Error('the')}            | ${Error}          | ${'the'}
-    ${new SyntaxError('quick')}    | ${SyntaxError}    | ${'quick'}
-    ${new EvalError('brown')}      | ${EvalError}      | ${'brown'}
-    ${new RangeError('fox')}       | ${RangeError}     | ${'fox'}
-    ${new ReferenceError('jumps')} | ${ReferenceError} | ${'jumps'}
-    ${new SyntaxError('over')}     | ${SyntaxError}    | ${'over'}
-    ${new URIError('the lazy')}    | ${URIError}       | ${'the lazy'}
-  `('returns false when not instance of $type with the correct error message', ({ error, type, errorMessage }) => {
-    expect(predicate(error, type, errorMessage)).toBe(true);
-  });
-});
-
-describe('toThrowWithMessage Predicate for a regex message and same type of error returns true', () => {
-  test.each`
-    error                          | type              | errorMessage
-    ${new Error('the')}            | ${Error}          | ${/the/}
-    ${new SyntaxError('quick')}    | ${SyntaxError}    | ${/quick/}
-    ${new EvalError('brown')}      | ${EvalError}      | ${/brown/}
-    ${new RangeError('fox')}       | ${RangeError}     | ${/fox/}
-    ${new ReferenceError('jumps')} | ${ReferenceError} | ${/jumps/}
-    ${new SyntaxError('over')}     | ${SyntaxError}    | ${/over/}
-    ${new URIError('the lazy')}    | ${URIError}       | ${/the lazy/}
-  `('returns false when not instance of $type with the correct error message', ({ error, type, errorMessage }) => {
-    expect(predicate(error, type, errorMessage)).toBe(true);
-  });
-});
-
-describe('toThrowWithMessage Predicate for a string message and same type of error returns false', () => {
```

Comment:
Does this test exist in _index.test.js_?

Reply:
See https://github.com/jest-community/jest-extended/pull/410/commits/dae7ff1ce8f9e9e6f27495a2c64d3de53d83c970